models/double_entry_transactions/int_quickbooks__sales_receipt_double_entry.sql

```diff
@@ -29,6 +98,42 @@ items as (
         and item.source_relation = parent.source_relation
 ),
 
+
+{% if var('using_sales_receipt_tax_line', False) %}
```

Comment:
when the following variables are set I run into a general ledger unique id test failure. Can you look into why that would be the case.
```yml
  using_sales_receipt_tax_line: true 
  using_tax_agency: true  
  using_tax_rate: false  
```

Reply:
The same happens for invoice and I imagine the others.

Reply:
Tax agency shouldn't be true when tax rate is false, so reconfigured the variables so that tax agency is only true when tax rate is enabled.